### ISSUE TITLE
TextInput Component doesn't disable click functionality when disabled

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30948]
## Description
When using a screen reader the TextInput component doesn't disable click functionality.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
0.63

**Comment by kacieb** [comment]
I think we can close this - should be the same issue as https://github.com/facebook/react-native/issues/30876

**Comment by lunaleaps** [comment]
@nadiia Can we close this per @kacieb's comment?

**Comment by lunaleaps** [comment]
Closing due to #30876
